contracts/modules/SecurityManager.sol

```diff
@@ -232,13 +231,13 @@ abstract contract SecurityManager is BaseModule {
      * @param _guardian The guardian to add.
      */
     function addGuardian(address _wallet, address _guardian) external onlyWalletOwnerOrSelf(_wallet) onlyWhenUnlocked(_wallet) {
-        require(!_isOwner(_wallet, _guardian), "SM: target guardian cannot be owner");
-        require(!isGuardian(_wallet, _guardian), "SM: target is already a guardian");
+        require(!_isOwner(_wallet, _guardian), "SM: _guardian cannot be owner");
```

Comment:
Can we avoid parameter names in the revert strings? It's un unnecessary dependency. We can just have _guardian -> guardian